src/main/resources/js/slackit-users.js

```diff
@@ -508,6 +530,18 @@ var SlackItClient = (function () {
 
     SlackItClient.prototype.onSlackItInvitePhaseInviteUser = function(self, user, channelId, channelName, nbGuests) {
         var _this = self;
+        var topic = AJS.params.baseURL + '/browse/' + _this.context.jiraIssueKey;
+        _this.rxClient.setTopicAsJiraLinkToChannelObservable(channelId, topic, _this.context.token, _this.context.slackApiBaseUrl).subscribe(
```

Comment:
minor suggestion: perhaps could be done when the channel is created initially, so any subsequent invite later (UI allows that) does not overwrite the channel topic (if modified by an user, e.g)
(onSlackItDialogDoStepCreateChannel?)